package.json

```diff
@@ -26,7 +26,7 @@
   },
   "dependencies": {
     "@justeat/f-logger": "0.7.2",
-    "@justeat/fozzie": "6.0.0-beta.0",
+    "@justeat/fozzie": "6.0.0-beta.7",
```

Comment:
I think this was the version I released that contained bad changes. `6.0.0-beta.8` is definitely fine.

Reply:
Interesting that I didn't have any errors, all worked as expected :grin: